Data/IDayOffRequestRepository.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using stunning_robot_HR.Models;
+
+namespace stunning_robot_HR.Data
+{
+    public interface IDayOffRequestRepository
```

Comment:
You're following the I of SOLID. A very focused concise interface here, well done.

Reply:
Thank you